CFW Builder integration

AidansLab opened the pull request:
### Description
Integrated Pip-Boy CFW Builder as a new page. There is one weird quirk where if the Pip-Boy is connected on one page, and then you go to the Builder page it cannot connect, though I'm not sure why.

### Changelog

- [Integrated CFW Builder as new page]

CodyTolene commented on src/app/pages/pip-boy-3000-mk-v-cfw-builder/pip-boy-3000-mk-v-cfw-builder-page.component.ts:
```diff
@@ -0,0 +1,210 @@
+
+    // Hook console.log to also send firmware write progress to terminal
+    // eslint-disable-next-line no-console
+    this.originalConsoleLog = console.log;
```
Resolved here
https://github.com/CodyTolene/pip-terminal/pull/172/commits/4de40c1adcfce2dc29cc92f2977a225f23122dc3

AidansLab commented:
Commit https://github.com/AidansLab/pip-terminal/pull/1 broke the fix I implemented for the serial connection glitch in the previous commit. I implemented a new one working around https://github.com/AidansLab/pip-terminal/pull/1 . Just for the future, the glitch happens when you connect on the CFW Builder page, then go to another page on the site, then come back to the CFW Builder page and try to connect again. For future testing this should be checked, since it is really annoying to implement a fix for.
